launcher/src/main/kotlin/com/cognifide/gradle/aem/launcher/ForkScaffolder.kt

```diff
@@ -102,6 +102,19 @@ class ForkScaffolder(private val launcher: Launcher) {
                             select(OpenMode.values().map { it.name.toLowerCase() }, OpenMode.ALWAYS.name.toLowerCase())
                         }
                     }
+                    group("Dispatcher") {
+                        define("dispatcherTarUri") {
+                            label = "Tar Archive URI"
+                            description = "Typically file named 'dispatcher-apache2.4-linux-x86_64-*.tar.gz'"
+                            text("https://download.macromedia.com/dispatcher/download/dispatcher-apache2.4-linux-x86_64-4.3.4.tar.gz")
+                        }
+                        define("dockerSafeVolumes") {
```

Comment:
this is no longer needed, clean it up;

it was reasonable when Docker Toolbox was not deprecated (now it is).